### PR TITLE
[HIRFUS-167] Vendors Page Validation

### PR DESCRIPTION
**Pull request by VincentNjele**
-In this PR
-I have put validation on the vendors page
VendorsDOMJS
VondorsDOMJS@

### CODE REVIEW

File: submissions/vincentNjele/Dom/vendors.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+addToCart.onclick = HandleFunction;
+
+function HandleFunction() {
+    
+    if(counter>0) {
+        
+        if(view<768){
+
+            addToCart.style.width = "150px";
```
Comment (ToluEnny24) [inline submissions/vincentNjele/Dom/vendors.js:28]: It is not good coding practice that your styles should be inside your html or js files

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Please remove your styles from your dom files

**Review by ToluEnny24 — APPROVED** [review]
Well done....

**Comment by VincentNjele** [comment]
Thank you so much.

**Pull request merged**